src/main/java/platform/codingnomads/co/springweb/gettingdatafromclient/requestbody/controllers/TaskController.java

```diff
@@ -48,4 +50,29 @@ public ResponseEntity<?> printMessage(@RequestBody(required = false) String mess
             return ResponseEntity.ok().body(message);
         }
     }
+
+    // Nathan practice
+    @PostMapping("/practice")
+    public ResponseEntity<?> snow(@RequestBody(required = false) String string) {
+        if (string == null){
+            string = "String is empty.";
+            return ResponseEntity.badRequest().body(string);
```

Comment:
Nit, It is unnecessary `string` assignment. 
```suggestion
            return ResponseEntity.badRequest().body("String is empty.");
```